internal/Panels/PopupDecorator.js

```diff
@@ -242,7 +251,7 @@ const PopupDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			delete rest.width;
 
 			return (
-				<Popup {...popupProps} className={className} data-index={index} id={id} css={css} no5WayClose noAlertRole={noAlertRole} noAnimation={noAnimation} onClose={onClose}>
+				<Popup {...popupProps} className={className} data-index={index} id={id} css={css} no5WayClose noAlertRole={noAlertRole} noAnimation={noAnimation} noOutline={noOutline} onClose={onClose}>
```

Comment:
I think we needs to add noOutline config in FixedPopupPanels. Could you please check it?

Reply:
The default value of noOutline config was wrong. I will fix this.